examples/quickstart/jupyter-notebooks/api-tutorial.ipynb

```diff
@@ -0,0 +1,442 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ad4e60b6",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Tutorial: Learn the basics of the Druid API\n",
+    "\n",
+    "<!--\n",
+    "  ~ Licensed to the Apache Software Foundation (ASF) under one\n",
+    "  ~ or more contributor license agreements.  See the NOTICE file\n",
+    "  ~ distributed with this work for additional information\n",
+    "  ~ regarding copyright ownership.  The ASF licenses this file\n",
+    "  ~ to you under the Apache License, Version 2.0 (the\n",
+    "  ~ \"License\"); you may not use this file except in compliance\n",
+    "  ~ with the License.  You may obtain a copy of the License at\n",
+    "  ~\n",
+    "  ~   http://www.apache.org/licenses/LICENSE-2.0\n",
+    "  ~\n",
+    "  ~ Unless required by applicable law or agreed to in writing,\n",
+    "  ~ software distributed under the License is distributed on an\n",
+    "  ~ \"AS IS\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY\n",
+    "  ~ KIND, either express or implied.  See the License for the\n",
+    "  ~ specific language governing permissions and limitations\n",
+    "  ~ under the License.\n",
+    "  -->\n",
+    "  \n",
+    "This tutorial introduces you to the basics of the Druid API and some of the endpoints you might use frequently, including the following tasks:\n",
+    "\n",
+    "- Checking if your cluster is up\n",
+    "- Ingesting data\n",
+    "- Querying data\n",
+    "- Deleting data\n",
+    "\n",
+    "In a Druid deployment, you have [Mastery, Query, and Data servers](https://druid.apache.org/docs/latest/design/processes.html#server-types) that all fulfill different purposes. The endpoint you use for a certain action is determined, partially, by which server governs that part of Druid and the processes that run on that server type. That's why the [API reference](https://druid.apache.org/docs/latest/operations/api-reference.html#historical) is organized by server type and process.\n",
+    "\n",
+    "## Table of contents\n",
+    "\n",
+    "- [Before you start](#Before-you-start)\n",
+    "- [Get basic cluster information](#Get-basic-cluster-information)\n",
+    "- [Ingest data](#Ingest-data)\n",
+    "- [Query your data](#Query-your-data)\n",
+    "- [Manage your data](#Manage-your-data)\n",
+    "- [Next steps](#Next-steps)\n",
+    "\n",
+    "For the best experience, use Jupyter Lab so that you can always access the table of contents."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8d6bbbcb",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Requirements\n",
+    "\n",
```

Comment:
Seems like it should go without saying, but do we tell people to install Jupyter?

Reply:
Needing either Jupyter lab or notebook is listed as one of the requirements

Reply:
I saw that later. Wondering if we'd include the install command? Up to you.

Reply:
I'll put the install command on the index page. Putting it in the notebook itself wouldn't make sense since they'd need to have it installed to see it.